test/unit/api/core/devicestate/deviceState.purge.test.js

```diff
@@ -0,0 +1,46 @@
+var should = require('should');
+var validateDeviceState = require('../../validator/deviceStateValidator.js');
+
+describe('DeviceState', function() {
+
+  describe('purge', function() {
+    
+    it('should return 2 devicestates only', function (done) {
+     	
+        var state = {
+            devicetype : 1,
+            value: 12.1,
+            datetime: '2010-01-01 22:00:00'
+        };
+        
+        gladys.deviceState.createByIdentifier('THIS_IS_MY_IDENTIFIER', 'test', 'binary', state)
+        .then(function(result){
+           options = {
+             devicetype : result.devicetype,
+             days : 365
+           }
+
+           return gladys.deviceState.purge(options);
+        })
+        .then(function() {
+           states = {
```

Comment:
Same here, use "var" when you create a variable. 

For simple tests like this you can maybe just do : 

```gladys.deviceState.get({devicetype: 1});```

So it's easier to read.